src/main/java/racing/CarGroup.java

```diff
@@ -0,0 +1,37 @@
+package racing;
+
+import static java.lang.System.*;
+
+import java.util.HashMap;
+
+/**
+ * 자동차 그룹 클래스
+ */
+public class CarGroup {
+    protected HashMap<Integer, Car> cars;
+    public StringBuilder stringBuilder;
```

Comment:
```suggestion
    protected final HashMap<Integer, Car> cars;
    public final StringBuilder stringBuilder;
```
객체의 안정성을 위해 불변을 설정해볼 수 있을 것 같아요 😄